src/config/logger.ts

```diff
@@ -22,9 +22,10 @@ const LOG_STREAM = filteringHttpLogsStream(FILTERING_FRONTEND_HTTP_LOGS_MIDDLEWA
 	bformat({ outputMode, levelInString: true })
 );
 
+// Todo existing config serializer is throwing occasional error, investigate.
 const responseSerializer = (res: AxiosResponse) => ({
 	...stdSerializers.res(res),
-	config: JSON.parse(util.inspect(res?.config)), // removes circular dependency in json
+	config: JSON.parse(inspect(res?.config)), // removes circular dependency in json
```

Comment:
wait, I'm fairly sure this is wrong.  I have some ideas on how to improve the logger here, might be simpler if we just go on a call I think.